rest-api-spec/src/yamlRestTest/java/org/elasticsearch/test/rest/ClientYamlTestSuiteIT.java

```diff
@@ -30,6 +30,7 @@ public class ClientYamlTestSuiteIT extends ESClientYamlSuiteTestCase {
         .module("mapper-extras")
         .feature(FeatureFlag.TIME_SERIES_MODE)
         .feature(FeatureFlag.DLM_ENABLED)
+        .nodes(2)
```

Comment:
I have two open PRs for the YAML cluster size and the YAML tests. Once merged, it will reduce the scope of this PR.